### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by gscott02** [saleem-hadad/larecipe#370]
If you run the Web Accessibility Evaluation Tool on a page generated in a fresh install of the package, there are several errors that crop up (e.g. empty links and low contrast). These are all important if building a fully accessible page for screen-reader users. I tried overriding the assets, but several of the issues are in assets not published by the package. Most of the fixes should be relatively straightforward. For example, the empty link in the LarecipeBackToTop.vue looks like this:

`  <div id="backtotop">
    <a href="#"></a>
  </div>`

A common approach to fix this might be:
`  <div id="backtotop">
    <a href="#"><span class="sr-only">Back to Top</span></a>
  </div>`

Is this something you are interested in fixing or potentially accepting a PR?
